fluentd-daemonset-azureblob.yaml

```diff
@@ -104,4 +104,4 @@ spec:
           path: /var/log
       - name: varlibdockercontainers
```

Comment:
We should change path names which are pointing out `/var/log/pods` instead of replacing paths.
These changes makes hard to debug and trace why `/var/log/pods` is mounted in `varlibdockercontainers`.